init-tools.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+__scriptpath="$(cd "$(dirname "$0")"; pwd -P)"
```

Comment:
In other scripts in the repo we tend to name this DIR and use the full version of this which will resolve through symlinks.

https://github.com/dotnet/cli/blob/rel/1.0.0/scripts/dockerrun.sh#L15